emergence_lib/src/terrain/mod.rs

```diff
@@ -35,12 +35,17 @@ impl TerrainType {
 
         builder.insert(*position);
         match self {
-            TerrainType::Plain => builder.insert(PlainTerrain),
-            TerrainType::Impassable => builder.insert(ImpassableTerrain {
-                impassable: Impassable,
-            }),
-            TerrainType::High => builder.insert(HighTerrain),
-        };
+            TerrainType::Plain => {
+                builder.insert(PlainTerrain);
+            }
+            TerrainType::Impassable => {
```

Comment:
we'll need to consider how to update this to allow for impassibility to be determined on a per-organism basis and not in the terrain gen/labeling. will be okay to merge for now but i'll go ahead and add an issue for this.

Reply:
@wallamide I'm not sure I totally understand, this is checking to see if `TerrainType::Rocky` (so it requires a rename too!)